core/os/thread.h

```diff
@@ -52,6 +52,11 @@ class Thread {
 
 	typedef uint64_t ID;
 
+	enum {
```

Comment:
I always wonder the reason for this way of declaring constants.

In any case, just to use the right type, better use `enum : ID`.

Reply:
Inertia I guess, but I can change it.